File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { N404ErrorHandlerSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await N404ErrorHandlerSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- N404ErrorHandler SDK exists test

local sdk = require("n404-error-handler_sdk")

describe("N404ErrorHandlerSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
